dev/breeze/doc/ci/05_workflows.md

```diff
@@ -269,7 +269,7 @@ Providers are skipped via selective checks (`skip-pre-commits` check).
 selective checks removing the "Providers" test type.
 
 `(9)` On non-main builds the integration tests for providers are skipped
-via `skip-provider-tests` selective check output.
+via `skip-providers-tests` selective check output.
```

Comment:
The more often I read this "provider" -> "providers" change... not native speaker... sounds a bit strange. Some native speaker around? Is a double-plural really correct?

Reply:
It does not, but it's everywhere already (including names of our packages). That ship has long sailed - it's been `bug`  nitially, but right now it became a `feature` I am afraid.

Like my Polish friend used to say "Better sh**ty but consistently" - in Polish "lepiej ch**owo ale jednakowo" - which is far more profane but also rhymes in Polish.